### ISSUE TITLE
Create a wiki page to document the desirable functionality of the Player module for future teams

### ISSUE THREAD

**Issue by afranzese99** [uchicago-cs/chiventure#1044]
Currently there does not exist documentation on the goals and expectations of the player module for future teams to work off of. To ensure future teams will have a substantial understanding of what needs to get done to continue to improve and implement the player module we will create documentation describing the current functionality and a list of what needs to get done/ what we would like to see get done. There does exists a [player interface implementation plan](https://github.com/uchicago-cs/chiventure/wiki/Player-~-Interface-Implementation-Plan) which consists of functions related to the player module interface (such as getters and setters that interact with other teams code like playerclass, skilltrees, quests, stats and effects etc.). However, most, if not all, of these functions should be implemented by the end of this sprint.

**Comment by WPDolan** [comment]
@afranzese99 and I have created the [Player Module Design Philosophy and Future Integration](https://github.com/uchicago-cs/chiventure/wiki/Player-~-Player-Module-Design-Philosophy-and-Future-Integration) wiki page which documents the current status of the player module while outlining what needs to be addressed in the future.

The wiki page includes the following major sections:
- "Why the Player Struct?" explains why the player module was created
- "The Player Struct" lists the current `player_t` struct as implemented in chiventure while also outlining the benefits that the struct provides the developers
- "The Player Module" explains the "wrapper" trend that the majority of functions in the player module achieve
- "Future Plans for the Player Module" outlines how the player module will need to be maintained by future iterations of the action-management/game-state team

**Comment by WPDolan** [comment]
Closing statement:

I have updated the wiki page to reflect changes merged to dev by Pull Request [#1123](https://github.com/uchicago-cs/chiventure/pull/1123). 

No additional changes to the wiki page were suggested by other members of the action-management team, so the wiki page as it stands should be sufficient for future teams to continue development of the player module. The page's four major sections, elaborated upon in the comment above, outline the contents of the wiki page created by this issue.
